table/sort.go

```diff
@@ -55,6 +55,10 @@ type rowsSorter struct {
 	sortedIndices []int
 }
 
+func (t *Table) SortedIndices() []int {
+	return t.sortedRowIndices
+}
+
```

Comment:
This should not be done as the value will not be available until something gets rendered. And another render might wipe it out as well.